comment/comment.go

```diff
@@ -27,7 +27,8 @@ type Comment struct {
 type Repository interface {
 	Create(ctx context.Context, u *Comment) error
 	Save(ctx context.Context, comment *Comment) (*Comment, error)
-	List(ctx context.Context, parent string) ([]*Comment, error)
+	List(ctx context.Context, parent string, start *int, limit *int) ([]Comment, uint64, error)
+	ListAll(ctx context.Context, parent string) ([]*Comment, error)
```

Comment:
Any reason to have two methods here? I think 1 List method that takes pagination arguments is enough.